tests/test_detect_target.py

```diff
@@ -93,39 +98,41 @@ def test_single_bus_image(self,
         """
         Bus image.
         """
-        # Setup
-        expected = cv2.imread(IMAGE_BUS_ANNOTATED_PATH)
-        assert expected is not None
-
         # Run
         result, actual = detector.run(image_bus)
+        detections = actual.detections
 
         # Test
         assert result
         assert actual is not None
+        assert detections is not None
     
-        error = rmse(actual, expected)
-        assert error < self.__IMAGE_DIFFERENCE_TOLERANCE
+        error = 0
```

Comment:
Do not accumulate error. Checks are to be done for each detection. For example: This test would pass when it should not if there are 100 detections where 1 of them has an error of 99.9x tolerance and the rest have 0 .